exchange-wrapper/migrations/3_initial_exchange_wrapper.js

```diff
@@ -0,0 +1,59 @@
+const { deployProxy } = require('@openzeppelin/truffle-upgrades');
+
+const ExchangeWrapper = artifacts.require('ExchangeWrapper');
```

Comment:
need to add this migration to deploy/ before deploying on main net

Reply:
agree, done

Reply:
add 20_initial_exchange_wrapper.js